src/sentry/models/team.py

```diff
@@ -290,7 +292,8 @@ def transfer_to(self, organization):
             # we use a cursor here to avoid automatic cascading of relations
             # in Django
             try:
-                cursor.execute("DELETE FROM sentry_team WHERE id = %s", [self.id])
+                with in_test_psql_role_override("postgres"):
+                    cursor.execute("DELETE FROM sentry_team WHERE id = %s", [self.id])
```

Comment:
Why is this necessary?

Reply:
This is to prevent `InsufficientPrivilege` in `test_team.TransferTest.test_existing_team`.
The `test_existing_team` test calls `team.transfer_to` which tries to delete a team via cursor.